pycm/pycm_obj.py

```diff
@@ -218,6 +218,26 @@ def __iter__(self):
         for key in self.matrix.keys():
             yield key, self.matrix[key]
 
+    def __contains__(self, class_name):
+        """
+        Check if the confusion matrix contains given class name.
```

Comment:
"Checking if the confusion matrix contains the given class name."

Reply:
fixed in https://github.com/sepandhaghighi/pycm/pull/476/commits/b8e1dc4c6168621d549dee146847de11011a8fcd

Reply:
Revert to `Check` in https://github.com/sepandhaghighi/pycm/pull/476/commits/2df3c5f502b8113a2a102ab70d44f80f6f0973a0 because of this error:
` D401: First line should be in imperative mood (perhaps 'Check', not 'Checking')`